bagitobjecttransfer/recordtransfer/forms/transfer_forms.py

```diff
@@ -566,85 +530,32 @@ def clean(self) -> dict:
         label=gettext("Title"),
     )
 
-    if not settings.USE_DATE_WIDGETS:
-        # Use _text to avoid jQuery input masks
-        start_date_of_material_text = forms.CharField(
-            min_length=2,
-            max_length=64,
-            required=True,
-            widget=forms.TextInput(attrs={"placeholder": gettext("e.g., 2000-03-14")}),
-            label=gettext("Earliest date"),
-            help_text=gettext(
-                "Enter the earliest date relevant to the files you're transferring."
-            ),
-        )
-        end_date_of_material_text = forms.CharField(
-            min_length=2,
-            max_length=64,
-            required=True,
-            widget=forms.TextInput(
-                attrs={
-                    "placeholder": "e.g., Fall 1925",
-                }
-            ),
-            label=gettext("Latest date"),
-            help_text=gettext("Enter the latest date relevant to the files you're transferring."),
-        )
-        start_date_is_approximate = False
-        end_date_is_approximate = False
-
-    else:
-        start_date_of_material = forms.DateField(
-            input_formats=[r"%Y-%m-%d"],
-            required=True,
-            widget=forms.DateInput(
-                attrs={
-                    "class": "start_date_picker reduce-form-field-width",
-                    "autocomplete": "off",
-                    "placeholder": "yyyy-mm-dd",
-                }
-            ),
-            label=gettext("Earliest date"),
-            help_text=gettext(
-                "Enter the earliest date relevant to the files you're transferring."
-            ),
-        )
-
-        # This field is intended to be tied to a button in a date picker for the start date
-        start_date_is_approximate = forms.BooleanField(
-            required=False,
-            widget=forms.CheckboxInput(
-                attrs={
-                    "hidden": True,
-                }
-            ),
-            label="hidden",
-        )
-
-        end_date_of_material = forms.DateField(
-            input_formats=[r"%Y-%m-%d"],
-            required=True,
-            widget=forms.DateInput(
-                attrs={
-                    "class": "end_date_picker reduce-form-field-width",
-                    "autocomplete": "off",
-                    "placeholder": "yyyy-mm-dd",
-                }
-            ),
-            label=gettext("Latest date"),
-            help_text=gettext("Enter the latest date relevant to the files you're transferring."),
-        )
+    date_of_materials = forms.RegexField(
+        regex=r"^(\d{4}-\d{2}-\d{2})(?:\s+-\s+(\d{4}-\d{2}-\d{2}))?$",
+        min_length=2,
+        max_length=64,
+        required=True,
+        widget=forms.TextInput(
```

Comment:
I think setting an error message that shows the expected format would be good to have here, as the current error message being displayed is a generic "Not valid".

```suggestion
        required=True,
        error_messages={
            "invalid": gettext('Date must be in format "YYYY-MM-DD" or "YYYY-MM-DD - YYYY-MM-DD"')
        },
        widget=forms.TextInput(
```

Reply:
Resolved in https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/331/commits/d21cc9b121477aca821055d731a07306ee4e6c9e